xml/tuning_storagescheduler.xml

```diff
@@ -125,99 +140,191 @@ noop deadline [cfq]</screen>
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term><filename>/sys/block/<replaceable>DEVICE</replaceable>/queue/iosched/low_latency</filename>
-     </term>
-     <listitem>
-      <para>
-       When enabled (which is the default on &productname;) the scheduler
-       may dynamically adjust the length of the time slice by aiming to meet
-       a tuning parameter called the <literal>target_latency</literal>. Time
-       slices are recomputed to meet this <literal>target_latency</literal>
-       and ensure that processes get fair access within a bounded length of
-       time.
-      </para>
-     </listitem>
-    </varlistentry>
-    <varlistentry>
-     <term><filename>/sys/block/<replaceable>DEVICE</replaceable>/queue/iosched/target_latency</filename>
-     </term>
-     <listitem>
-      <para>
-       Contains an estimated latency time for the
-       <systemitem class="resource">CFQ</systemitem>.
-       <systemitem class="resource">CFQ</systemitem> will use it to
-       calculate the time slice used for every task.
-      </para>
-     </listitem>
-    </varlistentry>
-     <varlistentry>
-       <term><filename>/sys/block/<replaceable>DEVICE</replaceable>/queue/iosched/group_idle_us</filename></term>
-       <listitem>
-         <para>To avoid starving of blkio cgroups doing dependent I/O, CFQ
-           waits a bit after completion of I/O for one blkio cgroup before
-           scheduling I/O for a different blkio cgroup. When
-           <literal>slice_idle_us</literal> is set, this parameter does not
-           have a big impact. However, for fast media, the overhead of
-           <literal>slice_idle_us</literal> is generally undesirable.
-           Disabling <literal>slice_idle_us</literal> and setting
-           <literal>group_idle_us</literal> is a method to avoid starvation
-           of blkio cgroups doing dependent I/O with lower overhead. Note that
-           the file <filename>group_idle</filename> controls the same tunable
-           however with millisecond granularity.
-         </para>
-       </listitem>
-     </varlistentry>
-   </variablelist>
+
+   <table xml:id="tab-tunables-cfq">
+     <title><systemitem class="resource">CFQ</systemitem> tunable parameters</title>
+     <tgroup cols="2">
+       <colspec colnum="1" colname="1" colwidth="3000*"/>
+       <colspec colnum="2" colname="2" colwidth="7000*"/>
+       <thead>
+	 <row>
+	   <entry><para>File</para></entry>
+	   <entry><para>Description</para></entry>
+	 </row>
+       </thead>
+       <tbody>
+	 <row>
+	   <entry><para><filename>slice_idle</filename></para></entry>
+	   <entry><para> When a task has no more I/O to submit in its
+	   time slice, the I/O scheduler waits for a while before
+	   scheduling the next thread. File
+	   <filename>slice_idle</filename> is the time in milliseconds
+	   the I/O scheduler waits.  Waiting for more I/O from a
+	   thread can improve locality of I/O. Additionally, it avoids
```

Comment:
I think the suggestion would not be correct: I would understand it as "the starving processes are not allowed to perform dependent I/O."
But what is meant here (my understanding): we want to avoid starvation of a process if the process is doing/using dependent I/O. With the slice_idle time we allow to submit all I/O requests that form a dependent I/O and thus avoid starvation of the process. Vice versa: without slice_idle time we can cause starvation of many processes (if they are using dependent I/O).
I agree that the original wording is not optimal.

Reply:
Ah, I see. Then let's keep the original wording.